cypress/e2e/reject-all.cy.js

```diff
@@ -0,0 +1,80 @@
+describe("Start to finish user journey for rejecting all outputs and downloading the empty zip", () => {
+  beforeEach(() => {
+    cy.visit("http://localhost:8000");
+  });
+
+  it("Rejects all files, visits the summary page, and downloads approved outputs", () => {
```

Comment:
Not sure about this, but maybe something like:
```suggestion
  it("Rejects all files, visits the summary page, and downloads review details", () => {
```